File: Clase 5 - Estructuras Condicionales/if ternario.py
# if ternario
edad = 19

# print("eres mayor de edad") if edad >= 18 else print("eres menor de edad")
# print( "eres mayor de edad" if edad >= 18 else "eres menor de edad")
# mensaje = "eres mayor de edad" if edad >= 18 else "eres menor de edad"

mensaje = f"eres {'mayor' if edad >= 18 else 'menor'} de edad"
print(mensaje)

File: Clase 5 - Estructuras Condicionales/if ternario.js
// if ternario

const edad = 17

/* edad >= 18
  ? console.log("Eres mayor de edad")
  : console.log("Eres menor de edad") */

/* console.log(edad >= 18
  ? "Eres mayor de edad"
  : "Eres menor de edad") */

/* const mensaje = edad >= 18
  ? "Eres mayor de edad"
  : "Eres menor de edad" */

const mensaje = `Eres ${edad >= 18 ? 'mayor' : 'menor'} de edad`
console.log(mensaje)
